vimrc

```diff
@@ -23,6 +23,33 @@ if has('nvim')
   let g:tern_request_timeout = 1
   let g:tern_show_signature_in_pum = 0
   set completeopt-=preview
+
+  " == neomake/neomake ==
+  let g:neomake_javascript_jshint_maker = {
```

Comment:
@zperrault `neomake` naming is pretty bad, but this one works for `eslint` too :)